pkg/controllers/updaterun/execution.go

```diff
@@ -33,6 +33,9 @@ var (
 	// stageUpdatingWaitTime is the time to wait before rechecking the stage update status.
 	// Put it as a variable for convenient testing.
 	stageUpdatingWaitTime = 60 * time.Second
+
+	// updateRunStuckThreshold is the time to wait on a single cluster update before marking update run as stuck.
+	updateRunStuckThreshold = 60 * time.Second
```

Comment:
this seems to be a bit too aggressive, a normal deployment with many replicas can take a few mins to get ready

Reply:
Let me set it as 5 minutes for now. It will not cause updateRun to stop. Just mark it in the status so users can investigate CRP and see if there's issue. If issue is fixed or recovered automatically, updateRun can continue and the status is removed. We also add metrics to track so that we could have a better understanding and treak it.